From 622ccdae7272d703f0027e2ccd5232744f83c224 Mon Sep 17 00:00:00 2001
From: Pan Xiuli <xiuli.pan@linux.intel.com>
Date: Mon, 23 Jul 2018 16:32:15 +0800
Subject: [PATCH 1836/2367] ASoC: SOF: uapi: ipc: refine dai index name

In sof_ipc_dai_config and sof_ipc_dai_config we both need DAI type and
index. Make the field name same and clear.

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>
---
 include/uapi/sound/sof-ipc.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/uapi/sound/sof-ipc.h b/include/uapi/sound/sof-ipc.h
index 14dafa7..c02264a 100644
--- a/include/uapi/sound/sof-ipc.h
+++ b/include/uapi/sound/sof-ipc.h
@@ -275,7 +275,7 @@ struct sof_ipc_dai_dmic_params {
 struct sof_ipc_dai_config {
 	struct sof_ipc_hdr hdr;
 	enum sof_ipc_dai_type type;
-	uint32_t id;	/* physical number if more than 1 of this type */
+	uint32_t dai_index; /* index of this type dai */
 
 	/* physical protocol and clocking */
 	uint16_t format;	/* SOF_DAI_FMT_ */
@@ -600,7 +600,7 @@ struct sof_ipc_comp_dai {
 	struct sof_ipc_comp comp;
 	struct sof_ipc_comp_config config;
 	enum sof_ipc_stream_direction direction;
-	uint32_t index;
+	uint32_t dai_index; /* index of this type dai */
 	enum sof_ipc_dai_type type;
 	uint32_t dmac_config; /* DMA engine specific */
 }  __attribute__((packed));
-- 
2.7.4

